### PR TITLE
Moves clap emote from carbon to human emotes

### PR DESCRIPTION
**Pull request by Burzah**
## What Does This PR Do
This relocates the clap emote from carbon emotes and moves it to human emotes. This prevents a runtime occurring when mobs without hands used the clap emote.
This also fixes https://github.com/ParadiseSS13/Paradise/issues/20419.
## Why It's Good For The Game
It cleans up the code a bit and prevents a runtime from occurring.
## Images
ApplicationFrameHost_Ht9CjFisNp
## Testing
Spawning multiple human and non human mobs and tested the clap emote.
Checked for runtime errors.
## Changelog
:cl:
fix: Non-humans can no longer attempt to clap
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human_emote.dm [hunk 1]
```diff
@@ -37,6 +37,41 @@
...
+			// well, we've got at least one
+			user.visible_message("[user] makes the sound of one hand clapping.")
+		return TRUE
+
+	return ..()
+
+/datum/emote/living/carbon/human/clap/get_sound(mob/living/user)
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		if(!H?.mind?.miming)
```
Comment (lewcc) [inline code/modules/mob/living/carbon/human/human_emote.dm:68]: this one's actually my fault
```suggestion
		if(!H.mind?.miming)
```

File: code/modules/mob/living/carbon/human/human_emote.dm [hunk 2]
```diff
@@ -37,6 +37,41 @@
...
+			to_chat(user, "<span class='warning'>You need arms to be able to clap.</span>")
+		else
+			// well, we've got at least one
+			user.visible_message("[user] makes the sound of one hand clapping.")
+		return TRUE
+
+	return ..()
+
+/datum/emote/living/carbon/human/clap/get_sound(mob/living/user)
+	if(ishuman(user))
```
Comment (lewcc) [inline code/modules/mob/living/carbon/human/human_emote.dm:66]: Can you make this an early return? This one's also my fault

File: code/modules/mob/living/carbon/human/human_emote.dm [hunk 3]
```diff
@@ -37,6 +37,41 @@
 	emote_type = EMOTE_VISIBLE
 	hands_use_check = TRUE
 
+/datum/emote/living/carbon/human/clap
+	key ="clap"
```
Comment (lewcc) [inline code/modules/mob/living/carbon/human/human_emote.dm:41]:
```suggestion
	key = "clap"
```

File: code/modules/mob/living/carbon/human/human_emote.dm (hunk 2 above)
Comment (Burzah) [reply code/modules/mob/living/carbon/human/human_emote.dm]: Fixed with the latest commit!

**Comment by Contrabang** [comment]
Can you combine the changelog into something like...

fix: Non-humans can no longer try to clap

Most people don't know what a runtime error is nor care about it, its not really player facing

**Comment by Henri215** [comment]
This is going to fix the issue brought up by https://github.com/ParadiseSS13/Paradise/issues/20419.

**Comment by Burzah** [comment]
> Can you combine the changelog into something like...
> 
> fix: Non-humans can no longer try to clap
> 
> Most people don't know what a runtime error is nor care about it, its not really player facing

You are absolutely right and I forgot about this. CL has been updated.

**Review by S34NW — APPROVED** [review]

**Pull request merged**